ui/src/components/layout/tracking-consent-modal.ts

```diff
@@ -60,33 +82,10 @@ export class NgmTrackingConsent extends LitElementI18n {
       padding-top: 24px;
     }
   `;
+}
 
-  render() {
-    return html`
-          <div class="header">
-            <h2>${i18next.t('tracking_header')}</h2>
-            <ngm-language-selector></ngm-language-selector>
-          </div>
-          <div class="main">
-            <h3>${i18next.t('tracking_limitations_of_liability_header')}</h3>
-            <p>${unsafeHTML(i18next.t('tracking_limitations_of_liability_text'))}</p>
-            <h3>${i18next.t('tracking_data_acquisition_header')}</h3>
-            <p>${unsafeHTML(i18next.t('tracking_data_acquisition_text'))}</p>
-              <ngm-core-checkbox .isActive="${this.isAllowed}"  @update="${() => this.isAllowed = !this.isAllowed}">${i18next.t('tracking_agree_label')}</ngm-core-checkbox>
-          </div>
-          <div class="footer">
-            <ngm-core-button @click="${() => this.accept()}">
-              ${i18next.t('accept')}
-            </ngm-core-button>
-          </div>
-      `;
-  }
+export type TrackingConsentModalEvent = CustomEvent<TrackingConsentModalEventDetail>;
 
-  private accept() {
-    this.dispatchEvent(new CustomEvent('confirm', {
-      detail: {
-        allowed: this.isAllowed
-      }
-    }));
-  }
+export interface TrackingConsentModalEventDetail {
+  allowed: boolean;

```

Comment:
Add `is` prefix for booleans.
```suggestion
  isAllowed: boolean;
```